src/locales/en/common.json

```diff
@@ -860,6 +860,15 @@
       "currencySettingsTitle": "{{currencyName}} Settings",
       "placeholder": "No settings for this asset"
     },
+    "accounts": {
+      "header": "Accounts",
+      "title": "Accounts",
+      "desc": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut nisl lorem.",
```

Comment:
can't merge this

Reply:
Give me a better temporary wording and I'll update